modules/cactus-web/src/Table/Table.tsx

```diff
@@ -138,118 +119,144 @@ const TableBodyBase: FunctionComponent<TableBodyProps> = (props) => {
+const TableRow = styled(TableRowBase)``
+
+type TableComponentType = StyledComponentType<TableProps> & {
+  Header: StyledComponentType<TableHeaderProps>
+  Cell: StyledComponentType<TableCellProps>
+  Row: StyledComponentType<TableRowProps>
+  Body: StyledComponentType<TableBodyProps>
+}
+
+const Table = styled(TableBase)`
+  ${(p) => p.fullWidth && 'min-width: 100%'};
+  border-spacing: 0;
   color: ${(p) => p.theme.colors.darkestContrast};
-  font-size: 15px;
-  font-weight: normal;
-  > svg {
-    vertical-align: text-top;
-    margin: 1px 10px;
-  }
+  ${(p) => p.theme.textStyles.small};
 
-  ${(p) => p.theme.mediaQueries.medium} {
-    width: calc(160px * 0.7125);
-  }
-  ${(p) => p.theme.mediaQueries.large} {
-    width: calc(160px * 0.875);
+  th {
+    font-weight: bold;
   }
 
-  ${(p) => p.theme.mediaQueries.extraLarge} {
-    width: 160px;
+  td,
+  th {
+    background-color: ${(p) => p.theme.colors.white};
+    border-top: ${(p) => border(p.theme, 'transparent')};
+    border-bottom: ${(p) => border(p.theme, 'transparent')};
+    :first-child {
+      border-left: ${(p) => border(p.theme, 'lightContrast')};
+    }
+    :last-child {
+      border-right: ${(p) => border(p.theme, 'lightContrast')};
+    }
   }
-`
 
-export const TableRow = styled(TableRowBase)`
-  position: relative;
-  margin: 4px 4px;
-  outline: 0;
-  display: flex;
-  background-color: ${(p) => p.theme.colors.white};
-  ${(p) => getShape(p.theme.shape)};
-  ${(p) => getBorder(p.theme.border)};
-  border-color: ${(p) => p.theme.colors.lightContrast};
-
-  & small {
-    color: ${(p) => p.theme.colors.darkestContrast};
+  tr:nth-of-type(even) {
+    td,
+    th {
+      background-color: ${(p) => p.theme.colors.lightContrast};
+    }
   }
 
-  :hover {
+  &&& tr:hover {
     cursor: pointer;
-    background-color: ${(p) => p.theme.colors.transparentCTA};
-    ${(p) => getBorder(p.theme.border)}
-    border-color: ${(p) => p.theme.colors.callToAction};
-    box-shadow: ${(p) => getBoxShadow(p.theme)};
-
+    th,
+    td {
+      border-color: ${(p) => p.theme.colors.callToAction};
+    }
   }
 
-  :nth-of-type(even) {
-    background-color: ${(p) => p.theme.colors.lightContrast};
-    ${(p) => getShape(p.theme.shape)};
-    ${(p) => getBorder(p.theme.border)};
-    border-color: ${(p) => p.theme.colors.mediumContrast};
-    :hover {
+  &&& tr:focus {
+    outline: 0;
+    td,
+    th {
       background-color: ${(p) => p.theme.colors.transparentCTA};
       border-color: ${(p) => p.theme.colors.callToAction};
     }
   }
-  :focus {
-    ::after {
-      content: '';
-      display: block;
-      position: absolute;
-      height: calc(100% + 10px);
-      width: calc(100% + 10px);
-      top: -5px;
-      left: -5px;
-      ${(p) => getBorder(p.theme.border)}
-      ${(p) => getDoubleBorder(p.theme.shape)};
-      border-color: ${(p) => p.theme.colors.callToAction};
-      box-sizing: border-box;
+
+  // first row
+  & > tr:first-of-type,
+  tbody > tr:first-child,
```

Comment:
Just noticed I missed something here, since tables are allowed to have more than one `tbody`.
```suggestion
  tbody:first-of-type > tr:first-child,
```

Reply:
Okay, that didn't fix it after all...

Reply:
This is kind of an uncommon use case, so I'd be fine if you leave it for now and I'll try to fix it in my PR.

Reply:
Yeah, I'm happy to leave it for now, but nice catch!